area_detector_handlers/handlers.py

```diff
@@ -126,7 +126,6 @@ def open(self):
 
     def close(self):
         super().close()
-        self._file.close()
```

Comment:
Can you leave a comment in the code about why we do not close the file so that some helpful contributor in the future does not go "I see a memory leak let me help!" and then break everything?

Should probably also add a test that we can create 2 instances of the handler, explicitly close one and the other still works.

Reply:
I took the liberty of adding the comment. I tried to write a test and I have been unable to reproduce the problem. See https://github.com/danielballan/area-detector-handlers/commit/b0786da303147b047577457fda926217cdc233e4. Any ideas why that doesn't raise?